scripts/variantstore/wdl/extract/hail_gvs_import.py

```diff
@@ -30,7 +30,7 @@ def import_gvs(argsfn, vds_path, references_path, temp_path):
     rg38.add_sequence(f'{references_path}/Homo_sapiens_assembly38.fasta.gz',
                       f'{references_path}/Homo_sapiens_assembly38.fasta.fai')
 
-
+    ## note that a full description of the import_gvs function written by Hail for this process can be found here: hail/python/hail/methods/impex.py
```

Comment:
what is the full path to the file?

Reply:
this is in an entirely different repo so I'm actually not sure how to reference it at all
it's here: https://github.com/hail-is/hail/pull/12017/files#

Reply:
if this file only exists as part of a branch, you can still reference it; in this case, the link would be https://github.com/tpoterba/hail/blob/import-gvs/hail/python/hail/methods/impex.py